### PR TITLE
Pattern menu work

### PR DESCRIPTION
**Pull request by rebmullin**
hey @scottnath  - still working on readme..but can you review the rest? thanks!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var fs = require('fs-extra');
+var twigger = require('pattern-importer').twigCompiler;
+
+function patternsMenu (src, obj, dest) {
```
Comment (rebmullin) [inline index.js]: hey @scottnath  - should i rename this too - objectToList - or something similar?

Comment (scottnath) [inline index.js]: agreed

**Comment by scottnath** [comment]
this all looks good. add the readme file though please.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+To create a list from an object
+
+
+## Usage
+
+in local index.js file, require the following:
+
+```
+var menuLister = require('./patterns-menu-list');
```
Comment (scottnath) [inline README.md]: change the name of the repo to `object-to-list` and change this too

**Comment by rebmullin** [comment]
hey @scottnath - updated. let me know what you think. thanks!

**Comment by scottnath** [comment]
@rebmullin merged!

**Pull request merged**